pages/Configuring/Environment-variables.md

```diff
@@ -40,6 +40,15 @@ environment on traditional Linux distros.
 - `HYPRLAND_NO_SD_NOTIFY=1` - If systemd, disables the `sd_notify` calls.
 - `HYPRLAND_NO_SD_VARS=1` - Disables management of variables in systemd and dbus activation environments.
 
+## Aquamarine Environment Variables <!-- ref https://github.com/hyprwm/aquamarine/blob/main/docs/env.md -->
+
+- `AQ_TRACE=1` - Enables more verbose logging.
+- `AQ_DRM_DEVICES=` - Set an explicit list of DRM devices (GPUs) to use. It's a colon-separated list of paths, with the first being the primary.
+  E.g. `/dev/dri/card1:/dev/dri/card0`
+- `AQ_NO_ATOMIC=1` - Disables drm atomic modesetting
```

Comment:
actually, please omit this

Reply:
done :)

Reply:
I meant only that one var (no_atomic)

Reply:
github is trolling lol
![image](https://github.com/user-attachments/assets/213562b2-11d0-46dc-9d7b-a2f703fa9b42)